recipes/ctng-compilers/yum_requirements.txt

```
@@ -0,0 +1,8 @@
+wget
+m4
+help2man
+patch
+gcc-gfortran
+gcc-c++
+rsync
```

Alternatively if you think the conda-forge dependencies of this should be minimised:
```suggestion
rsync
sed
findutils
make
```